src/app/api/ipfs/pinJson/route.ts

```diff
@@ -0,0 +1,56 @@
+import { NextRequest } from 'next/server'
```

Comment:
It felt too interdependent to explicitly allow juicecrowd.gg as an origin on juicebox.money for this endpoint. So, just copied it over here. Seems fine maybe